frontend/src/components/Title/Title.tsx

```diff
@@ -2,17 +2,17 @@ import React from 'react'
 import { makeStyles } from '@material-ui/core'
 import { spacing, colors } from '../../styling'
 
-export const Title: React.FC<{ online?: boolean }> = ({ children, online }) => {
+export const Title: React.FC<{ offline?: boolean }> = ({ children, offline }) => {
   const css = useStyles()
-  return <span className={css.title + (online ? ' online' : '')}>{children}</span>
+  return <span className={css.title + (offline ? ' offline' : '')}>{children}</span>
```

Comment:
you could use ` ' offline' || ''`

Reply:
Ah yes, if they were both the variable yes indeed, but in this case it's printing a string for the className that just happens to be the var name... so that wouldn't work in this case.